airmozilla/manage/archiver.py

```diff
@@ -145,7 +150,7 @@ def build_absolute_url(uri):
 
 
 def email_about_archiver_error(event, tag):
-    subject = "Unable to archive pending event with tag %s" % tag
+    subject = "Unable to archive process or pending event with tag %s" % tag
```

Comment:
Here and two places above, I think it should be "...processing or pending event..."